public/app/features/admin/AdminSettings.tsx

```diff
@@ -13,37 +14,21 @@ interface Props {
   navModel: NavModel;
 }
 
-interface State {
-  settings: Settings;
-  isLoading: boolean;
-}
-
-export class AdminSettings extends React.PureComponent<Props, State> {
-  state: State = {
-    settings: {},
-    isLoading: true,
-  };
-
-  async componentDidMount() {
-    const settings: Settings = await getBackendSrv().get('/api/admin/settings');
-    this.setState({
-      settings,
-      isLoading: false,
-    });
-  }
-
-  render() {
-    const { settings, isLoading } = this.state;
-    const { navModel } = this.props;
-
-    return (
-      <Page navModel={navModel}>
-        <Page.Contents isLoading={isLoading}>
-          <div className="grafana-info-box span8" style={{ margin: '20px 0 25px 0' }}>
-            These system settings are defined in grafana.ini or custom.ini (or overridden in ENV variables). To change
-            these you currently need to restart Grafana.
-          </div>
-
+const AdminSettings: React.FC<Props> = ({ navModel }) => {
```

Comment:
```suggestion
functino AdminSettings({ navModel }: Props) {
```

We should not use React.FC anymore it's considered bad practice (as it types the function in bad way), better to use implicit typing